### PR TITLE
bump: Raise coverage-parser version

### PR DESCRIPTION
**Pull request by franciscodua**
coverage-parser new version includes parsers for new coverage reports formats

### CODE REVIEW

File: src/main/scala/com/codacy/rules/ReportRules.scala [hunk 1]
```diff
@@ -127,12 +127,15 @@ class ReportRules(config: Configuration, coverageServices: => CoverageServices)
     val JacocoRegex = """(jacoco.*\.xml)""".r
     val CoberturaRegex = "(cobertura.xml)".r
```
Comment (josemiguelmelo) [inline src/main/scala/com/codacy/rules/ReportRules.scala]: Use escaped as in others: `\.`

File: README.md [hunk 2]
```diff
@@ -36,6 +36,17 @@ bash <(curl -Ls https://coverage.codacy.com/get.sh)
...
 
+This command assumes the coverage reports follow a name convention:
+```
+JaCoCo -> jacoco*.xml
+Cobertura -> cobertura.xml
+LCOV -> lcov.info | lcov.dat | *.lcov
+Clover -> clover.xml
+DotCover -> dotcover.xml
+OpenCover -> opencover.xml
+```
```
Comment (josemiguelmelo) [inline README.md]: Instead of code snippet, maybe bullet points? What do you think?

File: src/main/scala/com/codacy/rules/ReportRules.scala (hunk 1 above)
Comment (franciscodua) [reply src/main/scala/com/codacy/rules/ReportRules.scala]: Done

File: README.md (hunk 2 above)
Comment (franciscodua) [reply README.md]: I changed it. Not sure what the best approach is

**Pull request merged**